packages/uboot/rockchip/build.yaml

```diff
@@ -0,0 +1,14 @@
+requires:
+  - name: "toolchain"
+    category: "development"
+    version: ">=0"
+
+package_dir: /package
+
+prelude:
+- wget http://http.us.debian.org/debian/pool/main/u/u-boot/u-boot-rockchip_2025.01-2_arm64.deb -O u-boot.deb
```

Comment:
shouldn't there be a variable with the version here?

Reply:
Ideally we should move to build it ourselves if possible, so we got the most vanilla binaries out of this, but this was never updated so I just kept like that. Not even sure anyone is using it or even if we have one of these anymore lol